src/databricks/labs/ucx/queries/views/grant_detail.sql

```diff
@@ -29,4 +29,4 @@ SELECT
     catalog,
     database,
     table
-FROM $inventory.grants
\ No newline at end of file
+FROM $inventory.grants where database != split("$inventory",'[.]')[1]
```

Comment:
are you sure this is the right logic? $inventory won't have dots

Reply:
inventory comes with "hive_metastore.database_name" here. I tested the same.